src/library/trackcollection.cpp

```diff
@@ -510,4 +516,68 @@ StorageUnicodeFunctions::likeFunction(sqlite3_context *p,
                                     itrString, endString, E));
 }
 */
+//Private function
+
+bool TrackCollection::settingsPathStatus(){
+    struct stat sb; //this struct will hold the access info of the dir using struct here as below it has function of same name
```

Comment:
Vertical scrolling on GitHub is bad. Please break lines at column ~80 if there is no gut reason for don't do it. 
We use the space after // to distinguish comments from disabled code. In this case we need an space after //.